robber/matchers/__init__.py

```diff
@@ -5,12 +5,12 @@
     'called_once',
     'called_once_with',
     'called_with',
-    'contain'
+    'contain',
     'equal',
-    'ever_called'
+    'ever_called',
```

Comment:
No no, `called` and `ever_called` are 2 different things. `called` is equivalent to `mock.assert_called` while `ever_called` is equivalent to `mock.assert_any_call`